erts/emulator/beam/erl_process.h

```diff
@@ -78,7 +78,7 @@ typedef struct process Process;
 #define ERTS_MAX_NO_OF_DIRTY_CPU_SCHEDULERS ERTS_MAX_NO_OF_SCHEDULERS
 #define ERTS_MAX_NO_OF_DIRTY_IO_SCHEDULERS ERTS_MAX_NO_OF_SCHEDULERS
 
-#define ERTS_DEFAULT_MAX_PROCESSES (1 << 18)
+#define ERTS_DEFAULT_MAX_PROCESSES (1 << 20)
```

Comment:
Is there a particular reason why these are expressed as bit-shifts? If there _is_, then the shift count should be in a #define, if _not_, then the value should just be written out. The compiler's going to constant-fold it anyway, but it would forestall questions (like the one I'm asking right now).

Reply:
I think the reason why it is presented as a shift right now is:

> The actual maximum chosen may be much larger than the `Number` passed. Currently the runtime system often, but not always, chooses a value that is a power of 2. This might, however, be changed in the future.

In the documentation of `+P`. So right now it is always power of 2, but it is prepared for a case when this limitation will be lifted.

Reply:
Makes sense. Maybe copy the comment from erl_ptab.c: `/* A size that is a power of 2 is to prefer performance wise */`...?

Reply:
I think I'd prefer to have the minimal change merged for now, I'd be happy take a general cleanup of things like this in a separate PR. :-)